src/tests/pages/App.test.tsx

```diff
@@ -21,4 +21,4 @@ describe('App page', () => {
     expect(wrapper.find(AddPartyMember)).toHaveLength(1);
     expect(wrapper.find(Tip)).toHaveLength(1);
   });
-});
+});
```

Comment:
```suggestion
});

```
files should typically end with new line
https://stackoverflow.com/q/729692